### ISSUE TITLE
telegram implementation

### ISSUE THREAD

**Issue by mcpolo99** [bludnic/opentrader#86]
Hi! 

is there any plan for the future to implement telegram notifications?

**Comment by bludnic** [comment]
@mcpolo99 Yes, it's planned. Are you interested in just receiving events like order fulfillment, or also controlling the bots?

**Comment by mcpolo99** [comment]
> @mcpolo99 Yes, it's planned. Are you interested in just receiving events like order fulfillment, or also controlling the bots?

well i don't really know what i want tbh XD ,  for me personally i would say notifications is enough

**Comment by bludnic** [comment]
@mcpolo99 To stay updated on the progress, please join our community on Discord https://discord.gg/RS7y3ffvvG

**Comment by montymi** [comment]
It seems like the main issue with a Telegram implementation is that the site isn’t hosted publicly. To enable two-way communication, it needs to be a public-facing URL. 

You can use [Ngrok](https://ngrok.com) to temporarily publish the current version of your app and provide a public URL, which can then be connected to the OpenTrader Telegram bot.
